openedx/adg/lms/applications/forms.py

```diff
@@ -0,0 +1,115 @@
+"""
+Forms for custom registration form app.
+"""
+from datetime import date
+
+from dateutil.relativedelta import relativedelta
+from django import forms
+from django.core.validators import FileExtensionValidator, RegexValidator
+from django.utils.translation import ugettext_lazy as _
+
+from openedx.adg.lms.applications.models import UserApplication
+from openedx.adg.lms.registration_extension.models import ExtendedUserProfile
+from openedx.adg.lms.utils.date_utils import day_choices, month_choices, year_choices
+from student.models import UserProfile
+
+
+class ContactInformationForm(forms.Form):
+    """
+    Application Contact Information Form
+    """
+    name = forms.CharField(max_length=255, )
+    email = forms.EmailField()
+    city = forms.CharField(max_length=255, )
+    saudi_national = forms.BooleanField()
```

Comment:
We just define `saudi_national` property in simple form but aren't saving it in any model. Please verify this